File: asp-net-app/WebEMSOF/component/biz/Class_biz_emsof_request_return_comments.pas
unit Class_biz_emsof_request_return_comments;

interface

uses
  Class_db_emsof_request_return_comments;

type
  TClass_biz_emsof_request_return_comments = class
  strict private
    db_emsof_request_return_comments: TClass_db_emsof_request_return_comments;
  published
    //
    constructor Create;
    //
    function Bind
      (
      target: system.object;
      master_id: string
      )
      : boolean;
    procedure Log
      (
      master_id: string;
      demoter: string;
      content: string
      );
    //
  end;

implementation

constructor TClass_biz_emsof_request_return_comments.Create;
begin
  inherited Create;
  db_emsof_request_return_comments := TClass_db_emsof_request_return_comments.Create;
end;

function TClass_biz_emsof_request_return_comments.Bind
  (
  target: system.object;
  master_id: string
  )
  : boolean;
begin
  Bind := db_emsof_request_return_comments.Bind(target,master_id);
end;

procedure TClass_biz_emsof_request_return_comments.Log
  (
  master_id: string;
  demoter: string;
  content: string
  );
begin
  db_emsof_request_return_comments.Log(master_id,demoter,content);
end;

end.

File: trunk/emsi/asp-net-app/ConEdLink/component/biz/Class_biz_emsof_request_return_comments.cs
using System;
using Class_db_emsof_request_return_comments;
namespace Class_biz_emsof_request_return_comments
{
    public class TClass_biz_emsof_request_return_comments
    {
        private TClass_db_emsof_request_return_comments db_emsof_request_return_comments = null;
        //Constructor  Create()
        public TClass_biz_emsof_request_return_comments() : base()
        {
            db_emsof_request_return_comments = new TClass_db_emsof_request_return_comments();
        }
        public bool Bind(object target, string master_id)
        {
            bool result;
            result = db_emsof_request_return_comments.Bind(target, master_id);
            return result;
        }

        public void Log(string master_id, string demoter, string content)
        {
            db_emsof_request_return_comments.Log(master_id, demoter, content);
        }

    } // end TClass_biz_emsof_request_return_comments

}
